skbio/stats/distance/_permdisp.py

```diff
@@ -0,0 +1,269 @@
+    ------
+    TypeError
+        If, when using the spatial median test, the pcoa ordination is not of
+        type np.float32 or np.float64, the spatial median function will fail
+        and the centroid test should be used instead
+    ValueError
+        If the test is not centroid or median. test is set to median by default
+    TypeError
+        If the distance matrix is not an instance of a distance matrix
+
+    See Also
+    --------
+    permanova
+    anosim
+
+    Notes
+    -----
+    See [1]_ for the original method reference, as well as
+    ``vegan::betadisper``, available in R's vegan package [2]_.
+
+    References
+    ----------
+    .. [1] Anderson, Marti J. "Distance-Based Tests for Homogeneity of
+        Multivariate Dispersions." Biometrics 62 (2006):245-253
+
+    .. [2] http://cran.r-project.org/web/packages/vegan/index.html
+
+    Examples
+    --------
+    Load a 6x6 distance matrix and grouping vector denoting 2 groups of
+    objects:
+
+    >>> from skbio import DistanceMatrix
+    >>> dm = DistanceMatrix([[0,    0.5,  0.75, 1, 0.66, 0.33],
+    ...                       [0.5,  0,    0.25, 0.33, 0.77, 0.61],
+    ...                       [0.75, 0.25, 0,    0.1, 0.44, 0.55],
+    ...                       [1,    0.33, 0.1,  0, 0.75, 0.88],
+    ...                       [0.66, 0.77, 0.44, 0.75, 0, 0.77],
+    ...                       [0.33, 0.61, 0.55, 0.88, 0.77, 0]],
+    ...                       ['s1', 's2', 's3', 's4', 's5', 's6'])
+    >>> grouping = ['G1', 'G1', 'G1', 'G2', 'G2', 'G2']
+
+    Run PERMDISP using 99 permutations to caluculate the p-value:
+
+    >>> from skbio.stats.distance import permdisp
+    >>> import numpy as np
+    >>> #make output deterministic; not necessary for normal use
+    >>> np.random.seed(0)
+    >>> permdisp(dm, grouping, permutations=99)
+    method name               PERMDISP
+    test statistic name        F-value
+    sample size                      6
+    number of groups                 2
+    test statistic             1.03296
+    p-value                       0.35
+    number of permutations          99
+    Name: PERMDISP results, dtype: object
+
+    The return value is a ``pandas.Series`` object containing the results of
+    the statistical test.
+
+    To suppress calculation of the p-value and only obtain the F statistic,
+    specify zero permutations:
+
+    >>> permdisp(dm, grouping, permutations=0)
+    method name               PERMDISP
+    test statistic name        F-value
+    sample size                      6
+    number of groups                 2
+    test statistic             1.03296
+    p-value                        NaN
+    number of permutations           0
+    Name: PERMDISP results, dtype: object
+
+    PERMDISP computes variances based on two types of tests, using either
+    centroids or spatial medians, also commonly reffered to as a geometric
+    median or mediancentre. The spatial median is thought to yeild a more
```

Comment:
yeild -> yield

Reply:
Is `mediancentre` a term used in an implementation of this method? Would this more correctly be referred to in text as median center?

Reply:
I am bad at spelling. The term `mediancentre` was one I found reading through literature that supported this method. I don't think it is necessary to include, I can remove it.